pkg/cmd/server/kubernetes/node_auth.go

```diff
@@ -60,7 +60,8 @@ func (n NodeAuthorizerAttributesGetter) GetRequestAttributes(u user.Info, r *htt
 		APIVersion:   "v1",
 		APIGroup:     "",
 		Verb:         apiVerb,
-		Resource:     "nodes/proxy",
+		Resource:     "nodes",
+		Subresource:  "proxy",
```

Comment:
> actually, all the references to the joined resources in synthetic.go need review :-/

I renamed them to find them.  Interestingly enough, the authorization comparison still works because resource is opaque and still compares the entire thing.